libtfhe/core/arithmetic/torus_utils.h

```diff
@@ -119,7 +119,7 @@ class TorusUtils {
      * @return double value of the infinity norm
      */
     static double normInftyDist(const TORUS t1, const TORUS t2, const ZModuleParams<TORUS> *params) {
```

Comment:
this function should just be called distance
(norm infty is for vectors or polynomials)